File: Python/PRAK403-2210817220009-Azka Amalia.py
bil1 = int(input())
bil2 = int(input())
if bil1<bil2:
    i=bil1;j=bil2
    while i<=bil2 and j>=bil1:
       print("%d %d "%(i,j), end=' ')
       i+=1;j-=1
       if j==bil1-1: break
       else: print("-", end=' ')
else:
    i=bil1;j=bil2
    while i>=bil2 and j<=bil1:
        print("%d %d "%(i,j), end=' ') 
        i-=1;j+=1
        if j==bil1+1: break
        else: print("-", end=' ')

File: C/PRAK403-2210817220009-Azka Amalia.c
#include<stdio.h>
int main()
{
    int bil1, bil2, i, j;
    scanf("%d", &bil1);
    scanf("%d", &bil2);
    if(bil1<bil2){
        for(i=bil1,j=bil2; i<=bil2,j>=bil1; i++,j--){
            printf("%d %d ", i,j);
            if (j==bil1){
            break;}
            else{
                printf("-");}}}
    else{
        for(i=bil1,j=bil2; i>=bil2,j<=bil1; i--,j++){
            printf("%d %d ", i,j);
            if(j==bil1){
            break;}
            else{
                printf("-");}}}
}
